util/reggen/gen_dv.py

```diff
@@ -72,7 +98,39 @@ def gen_core_file(outdir: str,
         yaml.dump(core_data, core_file, encoding='utf-8')
 
 
-def gen_dv(block: IpBlock, dv_base_prefix: str, outdir: str) -> int:
+def get_dv_base_names_objects(dv_base_names: List[str]) -> Dict[str, DvBaseNames]:
+    ''' Convert the dv_base_names from string to a list of DvBaseNames objects.'''
+    dv_base_names_dict = {}  # type: Dict[str, DvBaseNames]
```

Comment:
Use defaultdict :)
```suggestion
    dv_base_names_dict = defaultdict(DvBaseNames)
```
With defaultdict, you dont need to check if key is present like you do on line 117.

Reply:
Oh wow, defaultdict is a cool keyword! Thanks for the suggestion!